docs-source/content/samples/simple/tanzu-kubernetes-service/_index.md

```diff
@@ -0,0 +1,763 @@
+  - `/tmp/mii-sample/model-images/model-in-image__WLS-v1/model.10.yaml`
+  - `/tmp/mii-sample/model-images/model-in-image__WLS-v1/model.10.properties`
+  - `/tmp/mii-sample/model-images/model-in-image__WLS-v1/archive.zip`
+
+If you don’t see the `weblogic-deploy.zip` file, then you missed a step in the prerequisites.
+
+Now, you use the Image Tool to create an image named `model-in-image:WLS-v1` that’s layered on a base WebLogic image. You’ve already set up this tool during the prerequisite steps.
+
+Run the following commands to create the model image and verify that it worked:
+
+If you are taking the JRF path through the sample, then remove --chown oracle:root from the imagetool.sh command below.
+
+```bash
+$ cd /tmp/mii-sample/model-images
+$ ./imagetool/bin/imagetool.sh update \
+  --tag model-in-image:WLS-v1 \
+  --fromImage container-registry.oracle.com/middleware/weblogic:12.2.1.4 \
+  --wdtModel      ./model-in-image__WLS-v1/model.10.yaml \
+  --wdtVariables  ./model-in-image__WLS-v1/model.10.properties \
+  --wdtArchive    ./model-in-image__WLS-v1/archive.zip \
+  --wdtModelOnly \
+  --wdtDomainType WLS \
+  --chown oracle:root
+```
+
+If you don’t see the `imagetool` directory, then you missed a step in the prerequisites.
+
+This command runs the WebLogic Image Tool in its Model in Image mode, and does the following:
+
+  - Builds the final Docker image as a layer on the `container-registry.oracle.com/middleware/weblogic:12.2.1.4` base image.
+  - Copies the WDT ZIP file that’s referenced in the WIT cache into the image.
+      - Note that you cached WDT in WIT using the keyword `latest` when you set up the cache during the sample prerequisites steps.
+      - This lets WIT implicitly assume it’s the desired WDT version and removes the need to pass a `-wdtVersion` flag.
+  - Copies the specified WDT model, properties, and application archives to image location `/u01/wdt/models`.
+
+When the command succeeds, it should end with output like the following:
+
+```
+[INFO   ] Build successful. Build time=36s. Image tag=model-in-image:WLS-v1
+```
+
+Also, if you run the `docker images` command, then you will see a Docker image named `model-in-image:WLS-v1`.
+
+> Note: If you have Kubernetes cluster worker nodes that are remote to your local machine, then you need to put the image in a location that these nodes can access. See [Ensuring your Kubernetes cluster can access images](#ensuring-your-kubernetes-cluster-can-access-images).
+
+
+#### Create WebLogic domain
+
+##### Deploy resources - Introduction
+
+In this section, you will deploy the new image to namespace `sample-domain1-ns`, including the following steps:
+
+- Create a namespace for WebLogic domain
+- Update WebLogic Kubernetes Operator to manage WebLogic domain namespace    
+- Create a Secret containing your WebLogic administrator user name and password.
+  - Create a Secret containing your Model in Image runtime encryption password:
```

Comment:
password:
If the next three bulleted items are sub-bulleted to this one, then they need to be indented further.